### DISCUSSION TITLE
Questions from a Linux Newbie

### DISCUSSION THREAD

**Discussion by codysechelski** [martinpl/tray-icons-reloaded#53] [Q&A] [answered]
Ok, so I have only been using Linux as my daily driver for a few weeks now. I've learned a lot but there are still huge holes in my knowledge that I'd like to fill. Sometimes my knowledge gap is so great that I don't even know the right questions to ask. So I thought I would ask them in screenshot form...

I am running ubuntu 20.04 LTS. I want to replicate something like what Windows does for the system tray icons. It looks like this extension is meant to do exactly that but until this morning I didn't even know what a GNOME extension was.

First I will share my settings in hopes that one of you kind internet strangers will be able to spot anything that might be wrong...

![2021-08-25_14-55](https://user-images.githubusercontent.com/1571418/130857018-aaf60f3a-a185-4d1c-aa71-7e45b03db997.png)

![2021-08-25_14-54](https://user-images.githubusercontent.com/1571418/130857048-d1233e56-992c-4d48-94f9-23ae2b78987f.png)

Ok, now, on to the questions.

Question 1:
Why do only some icons get nested into the "overflow" section? I have tried closing some of the apps and restarting them after I installed this extension but they refuse to jump inside the overflow menu.
![2021-08-25_14-46](https://user-images.githubusercontent.com/1571418/130857109-feaedf9d-4e25-4516-b141-2b3430bbc4d9.png)

Question 2:
This seems like it should be a simple fix but I don't see a setting for it. Why does my overflow section have a white background instead of a dark background as shown on the home page for this repo?
![2021-08-25_14-49](https://user-images.githubusercontent.com/1571418/130857383-629ff77d-b94f-45e2-9bdc-4c3baa8769c6.png)

Question 3:
There is one icon that looks fine when the extension is disabled but seems to disappear when I enable it. My guess is that it's a white raster icon and it's being shown on a white background. So maybe if I can fix the previously mentioned issue, it will take care of itself.
![2021-08-25_14-52](https://user-images.githubusercontent.com/1571418/130857551-902d065a-5370-4a25-bb22-fabfd7e62f04.png)

Thanks so much for indulging a newb like me to ask some questions to all of you Linux ninjas out there. I look forward to continuing my journey on the path to Linux ninja-hood 🙂

**Comment by martinpl** [comment, marked as answer]
Welcome on right side :wave: 
1 ) Ubuntu ships with own extension for tray icons, try to disable extension called 'AppIndicator and KStatusNotifierItem Support' or similar.
2, 3) You can change theme in GNOME Tweaks (last tab -> shell theme).

**Reply by codysechelski** [reply]
Thank you so much! It totally makes sense now.

Everything looks good now, except for some reason, when I activate this extension and turn off the built-in extension, only some of the icons are displayed. Any ideas why that might be?

Thanks!!

![2021-08-25_20-18](https://user-images.githubusercontent.com/1571418/130885006-03e6993a-7e34-48d0-bbe0-bd8f8420f6b4.png)
![2021-08-25_20-24](https://user-images.githubusercontent.com/1571418/130885016-3a5e4ef3-af5b-4945-857d-3fa69be1978c.png)

**Reply by martinpl** [reply]
I guess restart should do the job.

**Comment by codysechelski** [comment]
Well that was simple 🤦🏻‍♂️

Thanks again for your help. Everything is working perfectly now!
